internal/operations/operations.go

```diff
@@ -343,3 +344,27 @@ func checkDUT(dev binding.Device, op string) (*binding.DUT, error) {
 	}
 	return dev.(*binding.DUT), nil
 }
+
+
+// SwitchControlProcessor switchs to destination route processor
+func SwitchControlProcessor(ctx context.Context, dev binding.Device, dest string) error {
+       dut, err := checkDUT(dev, "restart routing")
```

Comment:
The second argument should be "switch control processor." Looks like a number of these arguments are screwed up elsewhere -- I'll fix the rest separately.